File: 945_minimum_increment_to_make_array_unique/minIncrementForUniqueSort.py
from ast import List


class Solution:
    def minIncrementForUnique(self, nums: List[int]) -> int:
        nums.sort()

        count = 0

        for i in range(0, len(nums) - 1):
            if nums[i] >= nums[i + 1]:
                count += nums[i] + 1 - nums[i + 1]
                nums[i + 1] = nums[i] + 1

        return count

File: 945_minimum_increment_to_make_array_unique/minIncrementForUniqueSort.ts
function minIncrementForUnique(nums: number[]): number {
  nums.sort((a, b) => a - b);

  let count = 0;

  for (let i = 0; i < nums.length - 1; i++) {
    if (nums[i] >= nums[i + 1]) {
      count += nums[i] + 1 - nums[i + 1];
      nums[i + 1] = nums[i] + 1;
    }
  }

  return count;
}
